lunes_cms/cms/models/document.py

```diff
@@ -25,8 +25,7 @@ class Document(models.Model):
     """
 
     id = models.AutoField(primary_key=True)
-    word_type = models.CharField(
-        max_length=255,
+    word_type = models.IntegerField(
         choices=Static.word_type_choices,
         default="",
```

Comment:
```suggestion
        default=Static.word_type_choices[0][0],
```
The default value should be one of the allowed choices.